src/libraries/System.Private.Xml/src/Resources/Strings.resx

```diff
@@ -1,17 +1,17 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <root>
-  <!--
-    Microsoft ResX Schema
-
+  <!-- 
```

Comment:
Something went wrong with the whitespace here during merge. Will send a followup commit to fix it.